XiEditor/Main.storyboard

```diff
@@ -39,6 +39,9 @@
                             <customView wantsLayer="YES" translatesAutoresizingMaskIntoConstraints="NO" id="q1P-kd-995" customClass="EditView" customModule="XiEditor" customModuleProvider="target">
                                 <rect key="frame" x="0.0" y="0.0" width="480" height="270"/>
                             </customView>
+                            <customView translatesAutoresizingMaskIntoConstraints="NO" id="x8W-KD-KOP" customClass="ShadowView" customModule="XiEditor" customModuleProvider="target">
```

Comment:
For me, I get a crash in ShadowView.setup because layer is nil, unless I also add `wantsLayer="YES"` here. Wonder why it seems to be working for you but not me.